### ISSUE TITLE
Namespacing non-ex commands

### ISSUE THREAD

**Issue by bovine3dom** [tridactyl/tridactyl#1124]
The `im_*` and `ex_*` (from #1123) ex commands are not really ex commands as typing them interactively in the command line won't work. We have other commands like this already - `editor`, for example.

The reason we're pretending they are ex commands is because we want to use the ex parser and macros.

I feel like they should probably be namespaced - `insert.move_cursor_left`, `ex.accept_string` - but that wouldn't work with the current macros, right?

**Comment by antonva** [comment]
Isn't it about time to revisit the macros anyhow? It makes sense to separate this logic from excmds.

**Comment by bovine3dom** [comment]
Yeah. I think this is probably part of #434. #1123 made me think about it again.

**Comment by bovine3dom** [comment]
Just keeping this thread up to date: we currently namespace them with `text` and `ex` but these aren't very easy to find, and we're still pretending they're ex-commands deep down.
